time/src/main/java/net/jqwik/time/internal/properties/arbitraries/DefaultOffsetTimeArbitrary.java

```diff
@@ -59,28 +62,50 @@
 			localTimes = localTimes.constrainPrecision(ofPrecision);
 		}
 
-		return localTimes.map(v -> OffsetTime.of(v, ZoneOffset.MIN));
+		if (offsetMin != null && offsetMax != null) {
+			zoneOffsets = zoneOffsets.between(offsetMin, offsetMax);
+		}
+
+		Arbitrary<OffsetTime> offsetTimes = Combinators.combine(localTimes, zoneOffsets).as(OffsetTime::of);
```

Comment:
Is it okay to use `Times.zoneOffsets().sample()` instead of `Combinators.combine` to combine value?

The problem is too heavy filtering and I don't know if it is possible to calculate good min/max values. Maybe with Times.zoneOffsets().sample() in the right range it is possible that there is not so much filtering... (See testcases)

Reply:
I currently cannot see the code in its context but never use sample() within an arbitrary. You lose all shrinking capabilities and may run into strange effects since sample() is made for use outside properties.

Reply:
Should I accept then that bad values runs into TooManyFilterMissesException? Or have you an idea what I can do?

Tomorrow I'll experiment with LocalTime min/max value maybe it's better then.